addons/forum-time-zones/addon.json

```diff
@@ -1,16 +1,58 @@
 {
-  "name": "Local timezone on forums",
-  "description": "Uses your local time zone for displaying post dates and times on the forums.",
+  "name": "Forum time changer",
+  "description": "Changes the times for posts in the forums.",
   "userscripts": [
     {
-      "url": "userscript.js",
+      "url": "timezones.js",
       "matches": ["topics"],
       "runAtComplete": false
+    },
+    {
+      "url": "matcher.js",
+      "matches": ["topics"],
+      "runAtComplete": false
+    }
+  ],
+  "credits": [
+    {
+      "name": "god286/webdev03",
+      "link": "https://github.com/webdev03"
+    },
+    {
+      "name": "_nix",
+      "link": "https://github.com/towerofnix"
     }
   ],
   "dynamicEnable": true,
   "dynamicDisable": true,
   "versionAdded": "1.0.0",
   "tags": ["forums", "community"],
+  "settings": [
+    {
+      "name": "Use local timezone",
+      "id": "local-timezone",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Use twelve hour time",
+      "id": "twelve-hour-time",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Use relative dates",
+      "id": "relative-dates",
+      "type": "boolean",
+      "default": true
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "The forum timezone changer connects to ScratchDB.",
+      "id": "scratchDBNotice"
```

Comment:
I don't think this is the case anymore, right?

Reply:
https://github.com/ScratchAddons/ScratchAddons/commit/386201165fd4f01bb57231f45d4fac30ea536057 changed it and I didn't notice